frame/bounties/src/benchmarking.rs

```diff
@@ -103,7 +103,7 @@ benchmarks! {
 		Bounties::<T>::propose_bounty(RawOrigin::Signed(caller).into(), value, reason)?;
 		let bounty_id = BountyCount::<T>::get() - 1;
 		Bounties::<T>::approve_bounty(RawOrigin::Root.into(), bounty_id)?;
-		Bounties::<T>::on_initialize(T::BlockNumber::zero());
+		Treasury::<T>::on_initialize(T::BlockNumber::zero());
```

Comment:
The benchmarking code was wrong because it called `on_initialize` of pallet-bounties instead of pallet-treasury. Only the `on_initialize` hook of pallet-treasury is defined and required to fund the bounty.